.github/workflows/container-deploy.yml

```diff
@@ -131,6 +131,24 @@ jobs:
           cd ${{ matrix.env}}
           make up
 
+    - name: Run integration tests
+      uses: appleboy/ssh-action@master
+      env:
+        DOCKER_CLIENT_TIMEOUT: 120
+        COMPOSE_HTTP_TIMEOUT: 120
+      with:
+        host: ${{ secrets.HOST }}
+        username: ${{ secrets.USERNAME }}
+        key: ${{ secrets.SSH_PRIVATE_KEY }}
+        proxy_host: ${{ secrets.PROXY_HOST }}
+        proxy_username: ${{ secrets.USERNAME }}
+        proxy_key: ${{ secrets.SSH_PRIVATE_KEY }}
+        envs: DOCKER_CLIENT_TIMEOUT,COMPOSE_HTTP_TIMEOUT
+        script_stop: false
+        script: |
+          cd ${{ matrix.env}}
+          make integration
```

Comment:
I personally don't think we should run integration test on the deployed server (for me integration tests are in the test pipeline…).
Most of the time integration tests are destructive, so you need to run them at least on a separate database for postgres / mongo etc.. and it's really risky to run them in a production environment.

If what you want to do is health check and or sanity checks we should use those names maybe.
At least we can have some test running in prod, but they should really be specific tests that are non destructive.